core/metacling/src/TCling.cxx

```diff
@@ -1340,9 +1349,15 @@ TCling::TCling(const char *name, const char *title, const char* const argv[])
       interpArgs.push_back(arg.c_str());
    }
 
+   // Add the Rdict module file extension.
+   cling::Interpreter::ModuleFileExtensions extensions;
+   EnvOpt = llvm::sys::Process::GetEnv("ROOTDEBUG");
```

Comment:
ROOTDEBUG is usually used to tweak the verbosity of ROOT and its numerical value is assigned to gDebug ....

Beside that is that a temporary or long term feature (that the extensions is updated depending on an environment variable)?

Reply:
It should be a long term feature which simplifies debugging rdict-related issues. What's the preferred way to do this?

Reply:
Using an environment variable is probably fine but you probably want to use a 'new' one with an explicit name ...